.github/workflows/CI.yaml

```
@@ -63,8 +63,8 @@ jobs:
       - name: Conda Environment Information
         shell: bash -l {0}
         run: |
-          conda info
-          conda list
+          mamba info
+          mamba list
```

```suggestion
          micromamba info
          micromamba list
```

Mamba apparently isn't installed in CI